### ISSUE TITLE
[cms] white screen changing selector on Visualization editor

### ISSUE THREAD

**Issue by cnavarreteliz** [Datawheel/canon#789]
If I change the viz type to Graphic before setting the api, I'm receiving a white screen.

```
TypeError: Cannot read property 'match' of undefined
    at e.exports (app.js:24602)
    at t.value (app.js:171083)
    at io (app.js:117904)
    at ro (app.js:117904)
    at Wo (app.js:117904)
    at Go (app.js:117904)
    at app.js:117904
    at Object.t.unstable_runWithPriority (app.js:124387)
    at Ls (app.js:117904)
    at Ps (app.js:117904)
```

**Comment by davelandry** [comment]
@jhmullen close this if fixed?

**Comment by jhmullen** [comment]
Yes, This was fixed in b351a011853db56da8df4e287f2c3a574e471a9d (by you lol)
